JotunnLib/Managers/ZoneManager.cs

```diff
@@ -26,7 +26,7 @@ private void Awake()
             Instance = this;
         }
 
-        public void registerAllToZNetScene(On.ZNetScene.orig_Awake orig, ZNetScene self)
+        private void RegisterAllToZNetScene(On.ZNetScene.orig_Awake orig, ZNetScene self)
```

Comment:
Could we rename this to be lowerCamelCase if it's private? Or make it internal and leave the casing?

Reply:
Nvm, my bad.